common.sh

```diff
@@ -130,11 +130,13 @@ build_install_ovmf()
 	pushd ovmf >/dev/null
 		run_cmd make -C BaseTools
 		. ./edksetup.sh --reconfig
+		touch OvmfPkg/AmdSev/Grub/grub.efi
 		run_cmd $BUILD_CMD
+		d=Build/AmdSev/DEBUG_GCC5/FV; dd if=$d/OVMF.fd of=$d/OVMF_CODE.fd skip=540672 bs=1; dd if=$d/OVMF.fd of=$d/OVMF_VARS.fd count=540672 bs=1
```

Comment:
How did you come up with the `540672` value?

Reply:
I don't think this is valid, anyway. The AmdSev package is specifically designed to NOT have a separate vars file, since the VMM/hypervisor controls that. The AmdSev package uses memory backed variable support which is not retained on shutdown of the guest.

Reply:
I removed this line and changed the copy commands after to only copy OVMF.fd to DEST.